Eplant/UI/LeftNav/index.tsx

```diff
@@ -60,12 +87,18 @@ export function LeftNav(props: {
           }
         }}
       />
-      <Collections
-        onSelectGene={props.onSelectGene}
-        selectedGene={props.selectedGene}
-      />
+        <Collections
+          onSelectGene={props.onSelectGene}
+          selectedGene={props.selectedGene}
+        /> </>}
+
+      {collapse && <>
+        <div onClick={() => setCollapse(false)} style={{ marginLeft: '10px' }}>
+          <SearchIcon sx={{ '&:hover': { cursor: 'pointer', transform: 'scale(1.1)' } }} />
+        </div>
+      </>}
       <Box flexGrow={1} />
-      <FormGroup>
+      {/* <FormGroup>
```

Comment:
Check with Nick on whether we're removing the toggle completely. If so, delete it rather than commenting it out.